src/main/java/com/github/alexdlaird/ngrok/protocol/CreateTunnel.java

```diff
@@ -69,6 +71,11 @@ public class CreateTunnel {
     private final String remoteAddr;
     private final String metadata;
     private final List<String> schemes;
+    private OAuth oauth;
+
+    public OAuth getOauth() {
```

Comment:
This should be below the constructor, below `getSchemes()` so its ordering matches.

Reply:
Done